### PR TITLE
Tesseract rebuild with newer libcurl

### PR DESCRIPTION
**Pull request by satmandu**
- this is dependent upon https://github.com/skycocker/chromebrew/pull/6120

Works properly:
- [x] x86_64

Builds properly:
- [x] x86_64
- [x] armv7l
- [ ] i686 (@uberhacker  build please?)

### CODE REVIEW

File: packages/tesseract.rb [hunk 1]
```diff
@@ -39,6 +38,7 @@ class Tesseract < Package
```
Comment (uberhacker) [inline packages/tesseract.rb:38]: Duplicate pango lines?

File: packages/tesseract.rb [hunk 2]
```diff
@@ -31,6 +29,7 @@ class Tesseract < Package
   depends_on 'glib'
```
Comment (uberhacker) [inline packages/tesseract.rb:29]: Duplicate cairo lines?

File: packages/tesseract.rb (hunk 1 above)
Comment (satmandu) [reply packages/tesseract.rb:37]: done

File: packages/tesseract.rb (hunk 2 above)
Comment (satmandu) [reply packages/tesseract.rb:28]: done

**Pull request merged**